transport/grpc/handler_test.go

```diff
@@ -64,3 +67,32 @@ func TestResponseWriter_Write(t *testing.T) {
 	assert.Equal(t, error(nil), err)
 	assert.Equal(t, strMsg, string(r.body.Bytes()))
 }
+
+func TestResponseWriter_AddHeaders(t *testing.T) {
```

Comment:
We don't generally add underscores to test names. Just use
`TestResponseWriterWithHeaders` or something similar.